src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -203,13 +204,18 @@ void initializeReference() {
      */
     void initializeReads() {
         if (! readArguments.getReadFiles().isEmpty()) {
-            SamReaderFactory factory = SamReaderFactory.makeDefault().validationStringency(readArguments.getReadValidationStringency());
+            final SamReaderFactory factory = SamReaderFactory.makeDefault().validationStringency(readArguments.getReadValidationStringency());
             if (hasReference()) { // pass in reference if available, because CRAM files need it
-                factory = factory.referenceSequence(referenceArguments.getReferenceFile());
+                factory.referenceSequence(referenceArguments.getReferenceFile());
```

Comment:
Recommend reverting to `factory = factory.referenceSequence()` out of paranoia that the return value might not be `this`